### PR TITLE
Update SPDX examples based on review feedback

### PR DESCRIPTION
**Pull request by goneall**
Resolves issue #400 
Resolves issue #401 
Resolves issue #409

### CODE REVIEW

**Comment by goneall** [comment]
Sorry about the horrible diffs - these are all machine generated

File: examples/SPDXYAMLExample-2.2.spdx.yaml [hunk 1]
```diff
@@ -1,392 +1,390 @@
...
+creationInfo:
+  comment: "This package has been shipped in source and binary form.\nThe binaries\
+    \ were created with gcc 4.5.1 and expect to link to\ncompatible system run time\
+    \ libraries."
+  created: "2010-01-29T18:30:22Z"
+  creators:
+  - "Tool: LicenseFind-1.0"
+  - "Organization: ExampleCodeInspect ()"
+  - "Person: Jane Doe ()"
+  licenseListVersion: "3.8"
```
Comment (tsteenbe) [inline examples/SPDXYAMLExample-2.2.spdx.yaml:13]: Minor issue but the license list is now at 3.9 should we update `licenseListVersion` in the examples?

Comment (goneall) [reply examples/SPDXYAMLExample-2.2.spdx.yaml]: Manually updated the license list to 3.9 (much faster than regenerating all the files).

**Comment by tsteenbe** [comment]
@goneall  Why does `packages` and not  `package` like RDF?

https://github.com/spdx/spdx-spec/blob/development/v2.2.1/examples/SPDXXMLExample-v2.2.spdx.xml#L246-L256 vs. https://github.com/spdx/spdx-spec/blob/development/v2.2.1/examples/SPDXRdfExample-v2.2.spdx.rdf.xml#L1821-L1831

**Pull request merged**